lib/migrations/20190530163257_track_typeOfTest.js

```diff
@@ -0,0 +1,31 @@
+const _ = require('underscore');
+
+exports.up = async function(knex, Promise) {
+    await knex.schema.table('students', function(t){
+        t.string('typeOfTest').after('stage').nullable();
```

Comment:
We can make it enum type either of online or offline

Reply:
how to make enum type in migration file

Reply:
I was telling in for constants.js my bad.